### DISCUSSION TITLE
Some square emojis like #️⃣, *️⃣, 1️⃣ cause unintended URL

### DISCUSSION THREAD

**Discussion by andhisan** [community/community#8654] [Other Feature Feedback, Questions, & Ideas]
## Emojis which cause unexpected behavior

I tried with several emojis like #️⃣, *️⃣, 1️⃣ and got these URLs.

|name|URL|
|---|---|
|`#️⃣ C#`|`%EF%B8%8F⃣-c`|
|`My number 1️⃣ list`|`my-number-1%EF%B8%8F⃣-list`|
|`*️⃣ foo`|`%EF%B8%8F⃣-foo`|

Expected behavior: emojis will simply be gone in the URL.

## Emojis which work correctly

|name|URL|
|---|---|
|`🔢 bar`|`bar`|
|`🆕 foo`|`foo`|

I also tried with similar square emojis like 🔢 or 🈁, but somehow they work correctly.

I'm not an expert on Unicode so I can't find the difference between these emojis. Maybe these are not recognized as emojis when they are converted to URLs.

**Comment by brianlovin** [comment]
Thanks for the heads up, we'll dig into this!
